code/apps/Managed Software Center/Managed Software Center/ja.lproj/Localizable.strings

```diff
@@ -1,8 +1,8 @@
 ﻿/* Multiple Updates message */
-"%s pending updates" = "%s個のアップデートを保留しています";
+"%@ pending updates" = "アップデートを%@個保留しています";
 
```

Comment:
Actually since Managed Software Update is a Cocoa-Python application, the string substitution is done in Python, and %s is the string placeholder, which is different from Objective-C's %@ object placeholder